src/assets/toolkit/scripts/lib/component/slideshow.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+import {u} from 'umbrellajs';
+export class Slideshow {
+  constructor (element, {
+    slideHolderSelector = '.js-Slideshow-slides',
+    slidesSelector = '.js-Slideshow-slide',
+    nextTriggerSelector = '.js-Slideshow-next',
+    prevTriggerSelector = '.js-Slideshow-prev',
+    currentCountElementSelector = '.js-Slideshow-current',
+    totalCountElementSelector = '.js-Slideshow-total',
+    classIsVisible = 'is-visible',
+    classWasVisible = 'was-visible',
+    classIsSlidingForward = 'is-sliding-forward',
+    classIsSlidingBack = 'is-sliding-back',
+    classIsForward = 'is-forward',
+    classIsBack = 'is-back'
+  } = {}) {
+
+    const slideHolder = u(slideHolderSelector, element);
+    const slides = u(slidesSelector, element);
```

Comment:
_Super_ picky...could we order these in the same order as the `Object.assign()`? Feel free to ignore this comment if you'd like. 😅  🙈 

```js
const slides = // code...
const slideHolder = // code...
```